tests/test_regression.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from cleanlab.regression import rank
+
+# To be used for all the tests
+labels = np.array([1, 2, 3, 4])
+predictions = np.array([1, 3, 4, 5])
+
+# test with deafault parameters
+def test_output_shape_type():
+    scores = rank.get_label_quality_scores(labels=labels, predictions=predictions)
+    assert labels.shape == scores.shape
```

Comment:
No need to fix the test, but I suppose multioutput datasets would fail this part of the test.

Reply:
@elisno we are not supporting multioutput regression yet, we have not even completed our functionality for scalar regression at the moment. We can revisit multioutput regression afterwards like we did for multilabel classification